homeassistant/components/proxmoxve/__init__.py

```diff
@@ -6,7 +6,7 @@
 from proxmoxer.backends.https import AuthenticationError
 from proxmoxer.core import ResourceException
 import requests.exceptions
-from requests.exceptions import ConnectTimeout, SSLError
+from requests.exceptions import ConnectTimeout, SSLError, ConnectionError
```

Comment:
Please address the lint failures.

Reply:
Please check the build to see what lint failures there are. You need to sort the imports and not overwrite built in exception names.